third_party/amd/python/test/test_extract_slice.py

```diff
@@ -39,7 +39,8 @@ def __str__(self):
     BlockedLayout([2, 2], [64, 1], [2, 2], [1, 0], [1, 1], [1, 1], [0, 1]),
     BlockedLayout([2, 2], [16, 4], [4, 1], [0, 1], [1, 1], [1, 1], [0, 1]),
     BlockedLayout([1, 8], [16, 4], [4, 1], [1, 0], [1, 1], [1, 1], [0, 1]),
-    BlockedLayout([1, 8], [16, 4], [4, 1], [0, 1], [1, 1], [1, 1], [0, 1]),
+    # FIXME(Lezcano): This layout errors out
+    #BlockedLayout([1, 8], [16, 4], [4, 1], [0, 1], [1, 1], [1, 1], [0, 1]),
```

Comment:
@antiagainst 
This layout is currently erroring out in the lowering. As @ThomasRaoux mentioned when reviewing the PR, I believe that the lowering will not be correct for a number of other layouts. In particular, I'm not sure it handles correctly broadcasting, but there can be other issues.